rwgame/DrawUI.cpp

```diff
@@ -104,8 +107,9 @@ void drawPlayerInfo(PlayerController* player, GameWorld* world,
 
     infoTextY += ui_textHeight;
 
-    if (player->getCharacter()->getCurrentState().health > ui_lowHealth
-        || fmod(world->getGameTime(), 0.5) >= .25) { // UI: Blinking health indicator if health is low
+    if ((world->state->hudFlash != HudFlash::FlashHealth
+        && player->getCharacter()->getCurrentState().health > ui_lowHealth)
+        || std::fmod(world->getGameTime(), 0.5f) >= .25f) { // UI: Blinking health indicator if health is low
```

Comment:
Based on [this](https://github.com/rwengine/openrw/issues/368#issuecomment-357550152) comment, the timer would be more accurate if it was ``fmod(t, 0.150f) <= 0.025f``.

Reply:
The map should flash about twice per second: https://youtu.be/klIcUiEw5jY?t=5m59s
The sum of shown and hiden durations should be about 500 ms. 
With fmod(t, 0.150f) the map will flash 6.7 times per second which is wrong.